### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by hcwinsemius**
- Bathymetry front end can be edited using a CSV format
- Bathymetry details view
- Lots of work on documentation
- Work on tutorial (needs to be finalized after this merge, some things don't work yet)

### CODE REVIEW

File: portal/controllers/bathymetry.py [hunk 1]
```diff
@@ -33,6 +85,66 @@ def bathymetry_coordinates(id):
...
+    crs_csv = read_epsg(f)
+    if crs_csv is None:
+        raise ValueError("No EPSG code found in header")
+    result = read_coords(f)
+    validate(instance=result, schema=schema)
+    bathymetry = Bathymetry.query.get(id)
+    # convert to site's EPSG code
+    crs_site = pyproj.CRS.from_epsg(bathymetry.site.position_crs)
+    transform = pyproj.Transformer.from_crs(crs_csv, crs_site,always_xy=True)
+    if not bathymetry:
```
Comment (RickHagenaars) [inline portal/controllers/bathymetry.py:104]: You're already passing bathymetry object into functions above, please put this check first.

File: portal/controllers/bathymetry.py [hunk 2]
```diff
@@ -33,6 +85,66 @@ def bathymetry_coordinates(id):
...
+    BathymetryCoordinate.query.filter(BathymetryCoordinate.bathymetry_id == bathymetry.id).delete()
+    for coordinate in result["coordinates"]:
+        x, y = transform.transform(coordinate['x'], coordinate['y'])
+        z = coordinate['z']
+        db.add(BathymetryCoordinate(x=x,y=y,z=z,bathymetry_id=bathymetry.id))
+
+    db.commit()
+    return jsonify(bathymetry.to_dict())
+
+@bathymetry_api.route("/api/bathymetry_details/<id>", methods=["POST"])
```
Comment (RickHagenaars) [inline portal/controllers/bathymetry.py:116]: You're using a POST but I don't really see any input body being provided, shouldn't this be a simple GET request?

File: portal/static/bathymetry_edit.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
+// TEMPORARY DOCUMENT READY FUNCTION
```
Comment (RickHagenaars) [inline portal/static/bathymetry_edit.js:1]: What's temporary about this ready function?

File: processing/tasks.py [hunk 4]
```diff
@@ -455,6 +457,10 @@ def run(movie, piv_kwargs={}, logger=logging):
         "http://portal/api/processing/run/%s" % movie["id"],
         json=Q,
     )
+    # requests.post(
```
Comment (RickHagenaars) [inline processing/tasks.py:460]: Commented code?

File: processing/tasks.py [hunk 5]
```diff
@@ -475,4 +481,8 @@ def run_camera_config(movie, logger=logging):
         "http://portal/api/processing/get_aoi/{:d}".format(movie['camera_config']["id"]),
         json=bbox_json,
     )
+    # requests.post(
```
Comment (RickHagenaars) [inline processing/tasks.py:484]: Commented code?

File: portal/controllers/bathymetry.py (hunk 1 above)
Comment (hcwinsemius) [reply portal/controllers/bathymetry.py:104]: Done

File: portal/controllers/bathymetry.py (hunk 2 above)
Comment (hcwinsemius) [reply portal/controllers/bathymetry.py:116]: Correct indeed, changed into GET

File: portal/static/bathymetry_edit.js (hunk 3 above)
Comment (hcwinsemius) [reply portal/static/bathymetry_edit.js:1]: removed

File: processing/tasks.py (hunk 4 above)
Comment (hcwinsemius) [reply processing/tasks.py:460]: this code is just for testing, I can quickly replace the post if I want to test locally (without docker)

File: processing/tasks.py (hunk 5 above)
Comment (hcwinsemius) [reply processing/tasks.py:484]: same here

**Pull request merged**